profiles/common/modules/custom/tmgmt_poetry/views/tmgmt_poetry.views_default.inc

```diff
@@ -0,0 +1,265 @@
+      'separator' => '',
+      'empty_column' => 0,
+    ),
+    'changed' => array(
+      'sortable' => 1,
+      'default_sort_order' => 'asc',
+      'align' => '',
+      'separator' => '',
+      'empty_column' => 0,
+    ),
+    'operations' => array(
+      'align' => '',
+      'separator' => '',
+      'empty_column' => 0,
+    ),
+    'source_language' => array(
+      'sortable' => 0,
+      'default_sort_order' => 'asc',
+      'align' => '',
+      'separator' => '',
+      'empty_column' => 0,
+    ),
+  );
+  $handler->display->display_options['defaults']['style_options'] = FALSE;
+  $handler->display->display_options['defaults']['row_plugin'] = FALSE;
+  $handler->display->display_options['defaults']['row_options'] = FALSE;
+  $handler->display->display_options['defaults']['relationships'] = FALSE;
+  /* Relationship: Translation Management Job Item: Tjid */
+  $handler->display->display_options['relationships']['tjid']['id'] = 'tjid';
+  $handler->display->display_options['relationships']['tjid']['table'] = 'tmgmt_job_item';
+  $handler->display->display_options['relationships']['tjid']['field'] = 'tjid';
+  $handler->display->display_options['defaults']['fields'] = FALSE;
+  /* Field: Translation Management Job: Reference */
+  $handler->display->display_options['fields']['reference']['id'] = 'reference';
+  $handler->display->display_options['fields']['reference']['table'] = 'tmgmt_job';
+  $handler->display->display_options['fields']['reference']['field'] = 'reference';
+  $handler->display->display_options['fields']['reference']['relationship'] = 'tjid';
+  $handler->display->display_options['fields']['reference']['exclude'] = TRUE;
+  /* Field: Translation Management Job: Target_language */
+  $handler->display->display_options['fields']['target_language']['id'] = 'target_language';
+  $handler->display->display_options['fields']['target_language']['table'] = 'tmgmt_job';
+  $handler->display->display_options['fields']['target_language']['field'] = 'target_language';
+  $handler->display->display_options['fields']['target_language']['relationship'] = 'tjid';
+  $handler->display->display_options['fields']['target_language']['label'] = 'Language';
+  /* Field: State */
+  $handler->display->display_options['fields']['state']['id'] = 'state';
+  $handler->display->display_options['fields']['state']['table'] = 'tmgmt_job_item';
+  $handler->display->display_options['fields']['state']['field'] = 'state';
+  $handler->display->display_options['fields']['state']['ui_name'] = 'State';
+  /* Field: Progress */
+  $handler->display->display_options['fields']['progress']['id'] = 'progress';
+  $handler->display->display_options['fields']['progress']['table'] = 'tmgmt_job_item';
+  $handler->display->display_options['fields']['progress']['field'] = 'progress';
+  $handler->display->display_options['fields']['progress']['ui_name'] = 'Progress';
+  $handler->display->display_options['fields']['progress']['label'] = 'Fields progress';
```

Comment:
Problem here is that in next column you have words count and it can be a bit misleading to leave just 'progress' as a label. I've made some small changes on tests.